include/dlaf/matrix/tile.h

```diff
@@ -454,6 +454,20 @@ pika::future<Tile<T, D>> splitTile(pika::future<Tile<T, D>>& tile, const SubTile
   return internal::createSubTile(old_tile, spec);
 }
 
+/// Create a writeable subtile of a given tile, without replacing given tile with the (full) tile
+/// that will come after the subtile will go out of scope.
+///
+/// The returned subtile will get ready, when the original tile was supposed to get ready.
+/// The next dependency in the dependency chain will become ready only when @p tile goes out of scope.
+template <class T, Device D>
+pika::future<Tile<T, D>> splitTileDiscard(pika::future<Tile<T, D>> tile, const SubTileSpec& spec) {
```

Comment:
You can hide the different function name need behind
```
pika::future<Tile<T, D>> splitTile(pika::future<Tile<T, D>>&& tile, const SubTileSpec& spec) {
  return splitTileDiscard(std::move(tile), spec);
}
```

Reply:
After some research with @rasolca we opted for not pursuing this road, since it would introduce an additional cost (compiler cannot apply copy-elision).

So, we would go for keeping the two separate functions with different names without any helper.